frappe/integrations/doctype/webhook/webhook.py

```diff
@@ -130,3 +131,10 @@ def get_webhook_data(doc, webhook):
 		data = json.loads(data)
 
 	return data
+
+
+def validate_json(string):
+	try:
+		json.loads(string)
+	except:
```

Comment:
```suggestion
	except TypeError:
```

Reply:
Added `ValueError` too for `JSONDecodeError`. `TypeError` is thrown only in case of `NoneType` inputs. `ValueError` is thrown for invalid JSONs